pkg/assets/stage.go

```diff
@@ -73,7 +71,19 @@ func Stage(dataDir string, name string, filemode os.FileMode) error {
 		return fmt.Errorf("failed to create dir %s: %w", filepath.Dir(p), err)
 	}
 
-	if ExecutableIsOlder(p) {
+	selfexe, err := os.Executable()
+	if err != nil {
+		logrus.Warn(err)
```

Comment:
If the error is returned anyways, wouldn't it be the caller's decision if and how this should be logged?